### DISCUSSION TITLE
`ADR Suggestion` Naming of EasyScience projects

### DISCUSSION THREAD

**Discussion by andped10** [easyscience/.github#1] [Ideas]
## Repo
Lib part `EasyProjectLib` CamelCase
GUI part `EasyProjectApp` CamelCase

## Lib python package
`easyproject` all small caps

## PyPI
`easyproject` when already taken something like: `easyscienceproject`

**Comment by rozyczko** [comment]
We should be checking the `pypi` repo and try to avoid situations like with `easyCore` -> `easyScienceCore`

**Comment by AndrewSazonov** [comment]
Right, we already ran into some problems with the EasyCore library because that name was already taken in PyPI, and we used `easysciencecore` as the package name, e.g. `pip install easysciencecore`, but imported it as `import easycore`. We solved this problem by renaming both `easycore` and `easysciencecore` to `easyscience`. To be consistent and not mislead users, I think it's important to have exactly the same names for both the library and the python package for all projects that will be published on PyPI.

**Comment by damskii9992** [comment]
I suppose this ADR suggestion should have been closed a while ago. Am I right in understanding that the ADR coming out of this simply states that our python packages should be named with all small caps? The discussion regarding repository names was extended quite a bit further in #20.

**Reply by rozyczko** [reply]
Correct. This content has been already covered elsewhere.
